### ISSUE TITLE
old build-depends

### ISSUE THREAD

**Issue by clockfort** [aristidb/random-extras#2]
Does random-extras actually need containers == 0.3.*? It'd be nice if it used a newer version, it's not playing well with other things for me.

```
cabal: cannot configure random-extras-0.18.1. It requires containers ==0.3.*
For the dependency on containers ==0.3.* there are these packages:
containers-0.3.0.0. However none of them are available.
containers-0.3.0.0 was excluded because vault-0.1.0.0 requires containers
==0.4.*
```

**Comment by aristidb** [comment]
Should be addressed with commit 0707a1463f5029a02bb936f69f23827c78ed3cb3

I also had to fix some other things to make it compile again. Didn't try to build the package in a while myself. :)

**Comment by aristidb** [comment]
I also released 0.19 with these fixes.

**Comment by clockfort** [comment]
Thanks, you rock :-)
